<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kitton Informática - Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Imagens/Logomarca Ícone.ico</normaloff>Imagens/Logomarca Ícone.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205, 240, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botaoLogar">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>710</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 0, 170);
color: rgb(255, 255, 255);
border-radius:40px;
}
QPushButton:hover{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>LOGAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:40px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="botaoNaoCad">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>820</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QCommandLinkButton{
color: rgb(0, 0, 255);
border-radius:20px;
}
QCommandLinkButton:hover{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Ainda não possui cadastro?</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="botaoEsqSenhaC">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>500</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QCommandLinkButton{
color: rgb(0, 0, 255);
border-radius:20px;
}
QCommandLinkButton:hover{
background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Esqueceu a senha?</string>
    </property>
   </widget>
   <widget class="QLabel" name="img6">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>90</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imagens/Logomarca.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imagens/Logomarca.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>351</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="cpf">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Email    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputEmail">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>450</y>
      <width>351</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="senha">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Senha   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputSenha">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
